### ISSUE TITLE
Syntax error for importing pysketcher composition

### ISSUE THREAD

**Issue by firasm** [rvodden/pysketcher#201]
I am trying to run the [`pendulum1` example](https://github.com/rvodden/pysketcher/blob/master/examples/pendulum1.py) and got an import error:

I believe the error is in Line 5 of the [_drawable.py](https://github.com/rvodden/pysketcher/blob/master/pysketcher/_drawable.py) file:

```
from pysketcher.composition._composition import Composition
```

This import should be:

```
from pysketcher.composition import Composition
```

Well, it solves the issue for me and I am no able to create sketches.

I'll submit a PR to fix this, would appreciate a review to make sure this doesn't break other things.

**Comment by rvodden** [comment]
That's exactly right - it really shouldn't be using the internal import. Even if its not causing your bug, can you re-open your PR please? Ill merge it. I've got a bit of a backlog of dependabot PRs so I'll probably merge them in this weekend and do another release.

**Comment by rvodden** [comment]
This is fixed by #202
